spec/01-lexical-syntax.md

```diff
@@ -9,17 +9,17 @@ chapter: 1
 Scala programs are written using the Unicode Basic Multilingual Plane
 (_BMP_) character set; Unicode supplementary characters are not
 presently supported.  This chapter defines the two modes of Scala's
-lexical syntax, the Scala mode and the _XML mode_. If not
+lexical syntax, the Scala mode, and the _XML mode_. If not
 otherwise mentioned, the following descriptions of Scala tokens refer
-to _Scala mode_, and literal characters ‘c’ refer to the ASCII fragment
+to _Scala mode_, and literal characters ‘c’ refers to the ASCII fragment
```

Comment:
I think "refer" is correct here. "characters" is plural. Perhaps a rewording would improve clarity.

Reply:
Yes. You are right. Changed it back.